recipes/libbpf/all/test_package/conanfile.py

```diff
@@ -0,0 +1,16 @@
+from conans import ConanFile, CMake, tools
+import os
+
+class LibbpfTestConan(ConanFile):
+    settings = "os", "compiler", "build_type", "arch"
+    generators = "cmake"
+
+    def build(self):
+        cmake = CMake(self)
+        cmake.configure()
+        cmake.build()
+
+    def test(self):
+        if not tools.cross_building(self.settings):
```

Comment:
```suggestion
        if not tools.cross_building(self):
```

Reply:
This is H062 btw:
https://github.com/conan-io/hooks/blob/5a4810657b358591c5805469f0dd2b78715efc4e/hooks/conan-center.py#L672-L684